package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1519,6 +1519,8 @@ def center_of_mass(group, wrap=False, unwrap=False, compound='group'):
            is deprecated and will be removed in version 3.0.
         """
         atoms = group.atoms
+        if not len(atoms):
+            raise ValueError("There are no atoms provided group")
```

Comment:
Assuming we are going with an error (@orbeckst?, no tests seem to complain) can you add a test for this error @jaclark5?  

Something like 

```python
with pytest.raises(ValueError, match ="there are no ..."):
    ...

```